src/RoadRegistry.BackOffice.Abstractions/Configuration/DynamoDbClientOptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace RoadRegistry.BackOffice.Abstractions.Configuration;
+
+using Amazon;
+
+public class DynamoDbClientOptions
+{
+    public const string ConfigurationKey = "DynamoDbClientOptions";
+    public string AccessKeyId { get; set; }
```

Comment:
AccessKeyId & AccessKeySecret won't be necessary

Reply:
> AccessKeyId & AccessKeySecret won't be necessary

This is required for the DynamoDbClient, same as distributed lock. Identical configuration also can be found inside the application definitions

Reply:
AWS ST takes care of this by giving the container the necessary AIM rights